### PR TITLE
Fix#137

### PR DESCRIPTION
**Pull request by BBallnus**
Issue #137 should be fixed

### CODE REVIEW

File: @PestoSamplingOptions/PestoSamplingOptions.m [hunk 1]
```diff
@@ -323,19 +323,25 @@
             
          else
             warning('No user-provided initial point found. Setting Initial points randomly.')
+            if size(par.min,1)==1
```
Comment (FFroehlich) [inline @PestoSamplingOptions/PestoSamplingOptions.m]: `par.min = par.min(:)` will cover far more cases. same for max.

**Comment by BBallnus** [comment]
Can you please be more specific in which case the current code lack proper initialization? Can you please give me an example option setting?

**Comment by FFroehlich** [comment]
```
options = PestoOptions()
getParameterSamples(...,...,options)
```
will fail because `options.PT.nTemps` is not set.

**Comment by FFroehlich** [comment]
what about ttps://github.com/ICB-DCM/PESTO/blob/cc6543b76a8892b2aaa1966e2d0dc59ad6750ee6/private/performPT.m#L115 

Is that check really doing what it is supposed to? I do not understand why you would expect size(theta0,1) ever to be 1 `unless parameters.number == 1`.

**Comment by BBallnus** [comment]
Yes, it does what it should do. This check is used for allowing the user to specify either a different initial chain position theta0 for every tempered chain sepererately, or using a single specification, which is applied to all the chains. I couldn't identfy any issues so far, when running the current code

**Pull request merged**